<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherWindow</class>
 <widget class="QMainWindow" name="TeacherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="teacher_info_tbtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>24</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/user.ico</normaloff>:/icons/user.ico</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>欢迎回来</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="teacher_name_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>老师</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>邮箱</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="teacher_email_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>181</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>我的课程</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="course_list"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="create_course_btn">
         <property name="text">
          <string>开课</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_course_btn">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>171</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>我的好友</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>170</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>消息列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="message_list"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="create_course_btn_3">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_course_btn_3">
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="BubblelTipToolButton" name="teacher_inform_tbtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>24</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/inform.png</normaloff>:/icons/inform.png</iconset>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>teacher_info_tbtn</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>teacher_inform_tbtn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BubblelTipToolButton</class>
   <extends>QToolButton</extends>
   <header>bubbleltiptoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
